Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/NotifyTransforms.java

```diff
@@ -52,9 +52,8 @@
     public LogEventRequestType transformNhinNotifyRequestToAuditMessage(LogNhinNotifyRequestType message) {
```

Comment:
This class can be removed. These are part of HIEM and that service implementation is removed from codebase.